runner/src/server/views/layout.html

```diff
@@ -34,7 +34,7 @@
       _paq.push(['trackPageView']);
       _paq.push(['enableLinkTracking']);
       (function() {
-        var u="https://web-analytics.fco.gov.uk/piwik/";
+        var u="{{ matomoUrl }}/piwik/";
```

Comment:
HI @bertrandmc - you need to change the noscript line too to be 

    <noscript><p><img src="{{ matomoUrl }}/piwik/piwik.php?idsite={{ matomoId }}&amp;rec=1" style="border:0;" alt="" /></p></noscript>

Reply:
Updated. Thanks for the head-up @trevorrea.

Reply:
Thanks for the head-up @trevorrea